lib/html.php

```diff
@@ -11,6 +11,21 @@
  * @link	https://github.com/rss-bridge/rss-bridge
  */
 
+/**
+ * Escape for html/xml context
+ */
+function e(string $s): string
```

Comment:
Just a note for future: PHP unfortunately cannot autoload stand-alone functions so we will need to require the file unconditionally, [here is how to do it with Composer](https://getcomposer.org/doc/04-schema.md#files). Or we can make it a static method in a class (I know, Java-ism, yuck). Then PHP will be able to require the file only when the class is used.